components/package.json

```diff
@@ -22,6 +22,7 @@
     "@stencil/core": "^1.8.8",
     "@stencil/redux": "^0.1.2",
     "@stencil/sass": "^1.1.1",
+    "@types/uuid": "^8.0.0",
```

Comment:
We probably don't need this anymore. If so, we should remove it.

Reply:
👍